website/docs/tasks/publish.md

```diff
@@ -0,0 +1,28 @@
+---
+title: publish
+---
+
+## Overview
+
+Taq `publish` is used to upload and publish files from a given directory to IPFS via Pinata. This can be used to upload contract or token metadata, image files, or any other file type supported by IPFS
+
+In order to use `publish`, you must have a valid API key and secret. You can get these from [Pinata](https://pinata.cloud/dashboard/api). For additional details, please see the [IPFS Pinata plugin](/docs/plugins/plugin-ipfs-pinata) documentation
```

Comment:
Also do you need an account to use that link to the pinata site? I clicked on it and I got a 404 page